### ISSUE TITLE
[jsx-indent] Bug in React props indentation

### ISSUE THREAD

**Issue by nwoltman** [yannickcr/eslint-plugin-react#2355]
This code is linted incorrectly:

```js
import cx from 'cx';

function Link({theme, className, ...otherProps}) {
  return (
    <a
      className={theme === undefined
        ? className
        : cx(
          'theme-' + theme,
          className,
        ) // jsx-indent rule says it expects 6 spaces here, which is wrong
      }
      {...otherProps}
    />
  );
}
```

**Comment by jzabala** [comment]
Tested this with the following configuration:

```js
"react/jsx-indent": ["error", 2]
```

It works as expected.
